recipes/cppfront/all/test_v1_package/pure2-hello.cpp2

```diff
@@ -0,0 +1,13 @@
+main: () -> int = {
+    std::cout << "Hello " << name() << "\n";
```

Comment:
Don't need to copy this file again, it can be removed. Test_v1_package should re-use it from test_package